WORKSPACE

```diff
@@ -153,6 +153,58 @@ load("//rust-deps/cxxbridge_crates:crates.bzl", cxxbridge_repositories = "crate_
 
 cxxbridge_repositories()
 
+# ========================================================================================
+# Node.js bootstrap
+#
+# workerd uses Node.js scripts for generating TypeScript types.
+
+http_archive(
```

Comment:
this will definitely make embedding workerd hard.

Maybe extract this to a separate function `setup_js_dependencies` in bzl file so that it can be called by downstream deps if needed?